addons/account/wizard/account_move_send.py

```diff
@@ -595,7 +621,7 @@ def _generate_invoice_documents(self, invoices_data, allow_fallback_pdf=False):
         }
         for invoice, invoice_data in invoices_data_pdf.items():
             if self._need_invoice_document(invoice) and not invoice_data.get('error'):
-                self._prepare_invoice_pdf_report(invoice, invoice_data)
+                self._prepare_invoice_pdf_report(invoice, invoice_data, self.pdf_template_id.xml_id)
```

Comment:
This function is also an api.model, from previous las comments i think this is wrong since self could be empty ? @smetl